common/argo/base/deployment.yaml

```diff
@@ -92,7 +92,7 @@ spec:
             fieldRef:
               apiVersion: v1
               fieldPath: metadata.namespace
-        image: argoproj/workflow-controller:v2.4.2
```

Comment:
@rushtehrani This isn't going to work I believe.
You need to change these values inside `kustomization.yaml`

Reply:
![image](https://user-images.githubusercontent.com/10407733/73994106-420e7b00-4909-11ea-9a81-d1dfc4f50fd5.png)
I switched to the branch and ran ./main generate. This was the output.

Reply:
Did you verify inside minikube?

Reply:
I did... did you run `cli init` again before running `deploy`?

Reply:
Yes, my cache files have deployment.yaml with the correct values.
Such as `        image: argoproj/argoui:v2.5.0-rc8`

Reply:
Let me try with another release and see.

Reply:
@aleksandrmelnikov you're right, it upgraded the configmap and a few other things but not the actual image.

Reply:
@aleksandrmelnikov ok, just pushed an update, it also removes `argo-ui` since that's not a service in the new version.